test/2.0/csv.spec.ts

```diff
@@ -722,11 +956,313 @@ test("validateRow wide conditionals", (t) => {
     "standard_charge | Payer Two | Special Plan | methodology": "",
     "additional_payer_notes | Payer Two | Special Plan": "",
   }
+
+  // If a "payer specific negotiated charge" is encoded as a dollar amount, percentage, or algorithm
+  // then a corresponding valid value for the payer name, plan name, and standard charge methodology must also be encoded.
+  // Since the wide format incorporates payer name and plan name into the column name, only methodology is checked.
+  const dollarWithMethodologyRow = {
+    ...basicRow,
+    "standard_charge | Payer One | Basic Plan | negotiated_dollar": "552",
+    "standard_charge | Payer One | Basic Plan | methodology": "fee schedule",
+  }
+  const dollarWithMethodologyErrors = validateRow(
+    dollarWithMethodologyRow,
+    5,
+    columns,
+    true
+  )
+  t.is(dollarWithMethodologyErrors.length, 0)
+  const dollarNoMethodologyRow = {
+    ...basicRow,
+    "standard_charge | Payer One | Basic Plan | negotiated_dollar": "552",
```

Comment:
I ran this test with the follow parameters:

```json 
const dollarNoMethodologyRow = {
     ...basicRow,
     "standard_charge | Payer One | Basic Plan | negotiated_algorithm": "",
     "standard_charge | Payer One | Basic Plan | negotiated_dollar": "",
     "standard_charge | Payer One | Basic Plan | negotiated_percentage": "",
     "standard_charge | Payer One | Basic Plan | methodology": "fee schedule",
   }
```
Should this trigger the `ITEM_REQUIRES_CHARGE` error?